docs/core/DelegationManager.md

```diff
@@ -51,6 +51,10 @@ Operators interact with the following functions to become an Operator:
 * [`DelegationManager.modifyOperatorDetails`](#modifyoperatordetails)
 * [`DelegationManager.updateOperatorMetadataURI`](#updateoperatormetadatauri)
 
+Once registered as an Operator, Operators can use an AVS's contracts to register with a specific AVS. The AVS will call these functions on the Operator's behalf:
+* [`DelegationManager.registerOperatorToAVS`](#registeroperatortoavs)
```

Comment:
In the context an AVS is using our middleware contracts, should we also specify that the "AVS" calling these functions refers to the RegistryCoordinator?